microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java

```diff
@@ -502,6 +502,17 @@ public int port() {
         return port;
     }
 
+    /**
```

Comment:
Adding a public API as part of an unrelated example. For future, this would deserve its own PR. Also please create a follow up to add tests for this new feature.

Reply:
I agree. Will add the test on a separate PR.